module4/projects/mod4Project/development.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: page
+title: Development
+---
+
+We will be using an Agile process during the development of this project. This will include:
+
+* Working in short 1 - 3 day "Sprints"
+* Using GitHub
+* Using a project management tool such as Github Projects
+* Conducting Story Grooming meetings
+* Doing daily Stand Ups
+* Demoing your project to your Project Manager
+* Holding Team Retros
+
+### git/GitHub
+
+* Commit Frequently
+* Decide on a [git workflow](https://www.atlassian.com/git/tutorials/comparing-workflows)
+* Make branches and submit Pull Requests for each new feature.
+    * DO NOT commit directly to main
+    * Keep your PRs as small as possible (one feature / user story per PR)
+    * Use screenshots and screencasts to demonstrate functionality.
+    * Your PR should be reviewed by at least one team member before it is merged.
+    * Tag your Project Manager if you have questions. DM your Project Manager letting them know you have tagged them.
+
+
+### Story Grooming
+
+* You should be using a project management tool to keep track of your stories.
+* When doing a "Story Grooming" you should:
+    * Make sure that the project board is up to date
+    * Look over the current backlog of stories and decide if you want to make any changes
+        * Are there any stories that need to be updated?
+        * Are there any stories that need to be added?
+    * Assign stories to individual team members or pairs
+        * Be clear on what it is that everyone should be doing
+* You should be Story Grooming every 1-2 days.
```

Comment:
What do you think about adding that they should start each sprint with Story Grooming?